akka-docs/rst/scala/code/docs/testkit/PlainWordSpec.scala

```diff
@@ -13,12 +13,10 @@ import org.scalatest.Matchers
 import org.scalatest.BeforeAndAfterAll
 
 //#implicit-sender
-class MySpec(_system: ActorSystem) extends TestKit(_system) with ImplicitSender
+class MySpec() extends TestKit(ActorSystem("MySpec")) with ImplicitSender
```

Comment:
Seems reasonable to me, as the sample does not show how to create an abstract base class for your tests, but an actual concrete test class